payments-core/src/main/java/com/stripe/android/model/LuxePostConfirmActionCreator.kt

```diff
@@ -19,7 +19,7 @@ sealed class LuxePostConfirmActionCreator {
         override fun create(stripeIntentJson: JSONObject): LuxePostConfirmActionRepository.Result {
             val returnUrl = getPath(returnToUrlPath, stripeIntentJson)
             val url = getPath(redirectPagePath, stripeIntentJson)
-            return if ((returnUrl != null) && (url != null)) {
+            return if ((url != null)) {
```

Comment:
Given the above limitation on the WebView vs. Chrome tabs and the result handling, there is currently code in place to make sure this path is not possible.